bin/vm-agent.js

```diff
@@ -98,11 +95,13 @@ function loadSysinfo(callback) {
 }
 
 
-async.waterfall([
+vasync.pipeline({funcs: [
```

Comment:
If we turn that into a `vasync.waterfall`, does it allow us to make `config`, `agentConfig` and `sdcConfig` local variables instead of global? That might be worth it.